src/screens/Services/Services.js

```diff
@@ -104,6 +100,11 @@ type Service = {|
   disabled?: boolean,
 |};
 
+type AddCashParam = {
```

Comment:
This type is definied also in AddCash screen. I believe we could remove it from here if we move add cash handling from onSubmit callback defined here to AddCash screen code (or some utils).